docs/core/definitions/units.rst

```diff
@@ -1,118 +1,4 @@
 Unit definitions and conversions
```

Comment:
should we port this file to markdown?

Reply:
Should be easy enough.

Reply:
Done now. I unfortunately have to use `eval-rst` to have the rST inside the doxygen comment render correctly.